account_multicompany_easy_creation/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Multicompany Easy Creation',
+    'summary': 'This module adds a wizard to create companies easily',
+    'version': '10.0.1.0.0',
+    'category': 'Uncategorized',
```

Comment:
We can make a new Category for all MC modules - Multicompany

Reply:
I agree `Multicompany` is good.